src/bioregistry/data/bioregistry.json

```diff
@@ -91014,7 +91014,91 @@
         "description": "Returns descriptors calculated from the molecule with RDKit",
         "homepage": "https://api.naturalproducts.net/latest/docs#/chem/get_descriptors_chem_descriptors_get",
         "name": "RDKit Descriptor via the Cheminformatics Microservice",
-        "uri_format": "https://api.naturalproducts.net/latest/chem/descriptors?format=json&toolkit=rdkit&smiles=$1"
+        "uri_format": "https://api.naturalproducts.net/latest/chem/descriptors?format=json&toolkit=$2&smiles=$1"
+      },
+      {
+        "code": "cms.depict2D",
+        "description": "Generates a 2D depiction of a molecule using CDK or RDKit with the given parameters.",
+        "homepage": "https://api.naturalproducts.net/latest/docs#/depict/Depict2D_molecule_depict_2D_get",
+        "name": "Depict 2D via the Cheminformatics Microservice",
+        "uri_format": "https://api.naturalproducts.net/latest/depict/2D?toolkit=$2&width=512&height=512&rotate=0&CIP=false&unicolor=false&smiles=$1"
+      },
+      {
+        "code": "cms.depict3D",
+        "description": "Generate 3D depictions of molecules using OpenBabel or RDKit.",
+        "homepage": "https://api.naturalproducts.net/latest/docs#/depict/Depict3D_Molecule_depict_3D_get",
+        "name": "Depict 3D via the Cheminformatics Microservice",
+        "uri_format": "https://api.naturalproducts.net/latest/depict/3D?smiles=$1&toolkit=$2"
+      },
+      {
+        "code": "cms.convertMol2D",
+        "description": "Generates 2D Coordinates using the CDK Structure diagram generator/RDKit/Open Babel and returns the mol block.",
+        "homepage": "https://api.naturalproducts.net/latest/docs#/convert/Create2D_Coordinates_convert_mol2D_get",
+        "name": "Generate 2D coordinates via the Cheminformatics Microservice",
+        "uri_format": "https://api.naturalproducts.net/latest/convert/mol2D?smiles=$1&toolkit=$2"
+      },
+      {
+        "code": "cms.convertMol3D",
```

Comment:
we need to keep all codes lowercase. maybe we can use snake_case to separate words instead. Also, `convert` doesn't imply direction, so maybe `cms.to_mol3d` or even `cms.mol3d` would be better